src/main/default/classes/EspressoMachineTest.cls

```diff
@@ -0,0 +1,54 @@
+@isTest
+class EspressoMachineTest {
```

Comment:
Same for this test class, try to not create the desired sut state using methods of the sut in your test methods. Instead try to set the target state either using a seperate ctor or by making fields test visisble so that they can be set by the test.